lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_bump_version_beta.rb

```diff
@@ -43,12 +42,6 @@ def self.details
 
       def self.available_options
         # Define all options your action supports.
-        [
-          FastlaneCore::ConfigItem.new(key: :app,
-                                       env_name: 'PROJECT_NAME',
-                                       description: 'The name of the app to get the release version for',
-                                       is_string: true), # true: verifies the input is a string, false: every kind of value
-        ]
       end
```

Comment:
I'm not sure. It's all a matter of taste but we've always kept that method around in the actions even if empty in all the other actions in the toolkit, for two reasons:

 - It makes it explicit when reading the action code that the action doesn't take any parameter
 - It makes it easier to add new ones when needed, always at the same position in the action files

Also, the method is generated for us when you create a new plugin via the appropriate fastlane command (in other words, it's part of the official template for creating fastlane actions)

At the end of the day it's probably mostly a matter of style, conventions and preferences, so I'm good with either but since we've kept those everywhere else in the toolkit codebase, I'd prefer to keep them there too for consistency.